libpcan/package.xml

```diff
@@ -1,26 +1,21 @@
-<package>
+<?xml version="1.0"?>
+<package format="2">
   <name>libpcan</name>
   <version>0.6.1</version>
   <description>This package wraps the libpcan to use it as a ros dependency</description>
-  <!-- <maintainer>Florian Weisshardt</maintainer> -->
 
   <license>LGPL</license>
 
   <url type="website">http://p103112.typo3server.info/fileadmin/media/linux/index.htm</url>
-  <!-- <url type="bugtracker"></url> -->
 
   <maintainer email="fmw@ipa.fhg.de">Florian Weisshardt</maintainer>
   <author>Florian Weisshardt</author>
 
   <buildtool_depend>catkin</buildtool_depend>
+
   <build_depend>libpopt-dev</build_depend>
+  <build_depend>linux-headers-generic</build_depend>
   <build_depend>mk</build_depend>
-  <build_depend>rosbuild</build_depend>
   <build_depend>rospack</build_depend>
-  <build_depend>roslib</build_depend>
-  <build_depend>linux-headers-generic</build_depend>
```

Comment:
I think we once agreed on using `depend` and `exec_depend` only...
`build_depend` only for `message_generation`
Thus, I think it is ok, if we declare them all `depend`